### ISSUE TITLE
Meta Box Custom Tweet counter needs to be replaced. 

### ISSUE THREAD

**Issue by ckmahoney** [warfare-plugins/social-warfare#617]
## Notes for testers

**What we expect**
Entering text into the Custom Tweet field will have the "Characters Remaining" counter update live. 

**How to Test**
In a post editor:

1. Scroll down to the Social Warfare meta box.
2. Find the "Custom Tweet" box. It should be third. 
3. Take note of how many characters you have remaining for the tweet. 
4. Enter text into the box. The "Characters Remaining" should decrease. 
5. Delete text from the box. The "Characters Remaining" should increase. 

If one of the above does not happen, please let us know as a comment to this thread. 

Once you have finished testing, please give this comment a thumbs up. 
### end notes

The metabox is missing the `X characters remaining` text. 

<img width="916" alt="screen shot 2018-08-28 at 1 42 57 pm" src="https://user-images.githubusercontent.com/26120849/44740407-5faaa480-aac8-11e8-8e29-4dd5e77bd610.png">

**Comment by ckmahoney** [comment]
The original issue was actually a result of "customTweet" needing to be renamed to "custom_tweet". However, after visiting the admin.js file, I noticed a couple sets of functions which needed to be reduced down to a single map and two methods.

**Comment by ckmahoney** [comment]
This issue has been marked as ready for testing.

**Comment by DustinWStout** [comment]
This is working for me. However it seems like the counter is off somehow. I'll have to do some more investigating, but it seems like Twitter is initially counting the URL and @mention characters in this custom tweet box even though Twitter says they say that they don't count characters for those items.

**Comment by nutsandbolts** [comment]
> This is working for me. However it seems like the counter is off somehow. I'll have to do some more investigating, but it seems like Twitter is initially counting the URL and @mention characters in this custom tweet box even though Twitter says they say that they don't count characters for those items.

Yes to this. A customer has reported this as well (iref 235077)

**Comment by DustinWStout** [comment]
It's just hard to guage because the Twitter share box no longer displays the character count. So maybe we just need to (as before) subtract the number of characters in the URL and username.

**Comment by iHelpers** [comment]
This is probably a new issue, but with 3.3.91 Pro installed, my post editor (legacy) is now unusable.  I am not able to open/expand the SW Meta Box and the entire page becomes unresponsive.  The only way to make is usable again is to disable SW Pro.. Re-actiavting it causes the same issue.

**Comment by ckmahoney** [comment]
@iHelpers 
I just tried to replicate by following these steps on a blank WP install:

Upload 3.3.91 zips for Core and Pro
Activate both plugins
Enter Pro license Key
Go to post editor 
Use SW metabox

And I was able to use the metabox fine. 

Note that Gutenberg is  was not installed at the time of replicating. 

Then I installed and activated Gutenberg, returned to the post editor, and it still functioned fine. 

*However*, we have noticed that when a post editor is trying to load new code (e.g. a plugin update), it will lock/hang/freeze the entire tab is unusable in Chrome.
In Firefox, we've noticed that it simply does not load the new code without forcing the browser to empty its cache.
